### PR TITLE
Squasheddaeobjexport

### PR DESCRIPTION
**Pull request by lonelyhawk**
exporting both .dae and .obj files simultaneously squashed the model
image
image

### CODE REVIEW

**Comment by Kjasi** [comment]
Looks good to me!

File: CgfConverter/Renderers/Wavefront/Wavefront.cs [hunk 1]
```diff
@@ -100,8 +100,7 @@ public override void Render(String outputDir = null, Boolean preservePath = true
                             }
 
                             // TODO: Transform Root Nodes here?
-
-                            file.WriteLine("o {0}", node.Name);
+                            //file.WriteLine("o {0}", node.Name);
```
Comment (Markemp) [inline CgfConverter/Renderers/Wavefront/Wavefront.cs:103]: Can you delete unneeded code & comments?

File: CgfConverter/Renderers/Collada/COLLADA.cs [hunk 2]
```diff
@@ -1407,6 +1401,12 @@ private Grendgine_Collada_Node CreateJointNode(CompiledBone bone)
                 localTransform.M21, localTransform.M22, localTransform.M23, localTransform.M24,
                 localTransform.M31, localTransform.M32, localTransform.M33, localTransform.M34);
 
+            //matrixValues.AppendFormat("{0:F6} {1:F6} {2:F6} {3:F6} {4:F6} {5:F6} {6:F6} {7:F6} {8:F6} {9:F6} {10:F6} {11:F6} {12:F6} {13:F6} {14:F6} {15:F6}",
+            //   bone.LocalTransform.M11, bone.LocalTransform.M12, bone.LocalTransform.M13, bone.LocalTransform.M14,
+            //   bone.LocalTransform.M21, bone.LocalTransform.M22, bone.LocalTransform.M23, bone.LocalTransform.M24,
+            //   bone.LocalTransform.M31, bone.LocalTransform.M32, bone.LocalTransform.M33, bone.LocalTransform.M34,
+            //   bone.LocalTransform.M41, bone.LocalTransform.M42, bone.LocalTransform.M43, bone.LocalTransform.M44);
+
```
Comment (Markemp) [inline CgfConverter/Renderers/Collada/COLLADA.cs:1409]: Can you remove old unneeded code please? :)

**Review by Markemp — CHANGES_REQUESTED** [review]
Just 2 small code cleanup items.  Thank you!

**Pull request merged**